src/services/dataService.js

```diff
@@ -772,6 +772,7 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
       this._watchInFlight(key, false);
       this._watchOptions(key, null);
     }
+    handle = null;
```

Comment:
Hey @benjaminapetersen I don't think it makes much difference here since `handle` is local to the function, so it goes out of scope immediately.

Reply:
aye let me check that.